src/ecs/resources/input.rs

```diff
@@ -93,4 +148,41 @@ impl InputHandler {
     pub fn keys_once(&mut self, keys: &[VirtualKeyCode]) -> bool {
         keys.iter().any(|key| self.key_once(*key)) && self.keys_down(keys)
     }
+
+    /// Returns an iterator for all the pressed down mouse buttons
+    pub fn pressed_mouse_buttons(&self) -> PressedMouseButtonsIterator {
+        PressedMouseButtonsIterator { iterator: self.pressed_mouse_buttons.keys() }
+    }
+
+    /// Check if `button` is currently pressed
+    pub fn mouse_button_down(&self, button: MouseButton) -> bool {
+        self.pressed_mouse_buttons.contains_key(&button)
+    }
+
+    /// Check if all `buttons` are currently pressed
```

Comment:
Should be `/// Checks` for consistency with the general documentation style, also missing a period.